content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -457,6 +458,18 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `ContextualLogging`: When you enable this feature gate, Kubernetes components that support
    contextual logging add extra detail to log output.
 - `CronJobTimeZone`: Allow the use of the `timeZone` optional field in [CronJobs](/docs/concepts/workloads/controllers/cron-jobs/)
+- `CRDValidationRatcheting`: Enable updates to custom resources to contain 
+   violations of their OpenAPI schema if the offending portions of the resource
+   update did not change. Allows users to update to stricter versions of a CRD
+   schema without bumping the version of the object or breaking workflows.
+   While most are supported, does not support ratcheting updates across all 
+   schema changes. The following OpenAPIV3 schema validations are not supported
+   by ratcheting under the current implementation and if violated will continue
+   to throw an error as normally:
+   - `allOf`/`oneOf`/`anyOf`/`not` nested validations
+   - `x-kubernetes-validations`
+   - `x-kubernetes-list-type`
+   - `required` fields
```

Comment:
Please make this shorter and then link to the relevant section of the documentation about CustomResourceDefinitions (which you need to update).